### ISSUE TITLE
Move defaults to declaration

### ISSUE THREAD

**Issue by DavidS** [ddfreyne/cri#52]
Please add a way to specify the default value for an option in a way that gets exposed to `--help`, e.g.:

```ruby
optional nil, :source, 'The source repository to load templates from.', default: 'git@github.com:puppetlabs/modulesync_configs.git'
```

```text
options:

    -h --help      show help for this command
       --summary   The source repository to load templates from.
                   Default value: git@github.com:puppetlabs/modulesync_configs.git
```

instead of 

```ruby
optional nil, :source, 'The source repository to load templates from.'
run do |opts, args, cmd|
  source = opts.fetch(:source, 'git@github.com:puppetlabs/modulesync_configs.git')
```

**Comment by ddfreyne** [comment]
Solid idea! I’ve implemented this in #55.

**Comment by ddfreyne** [comment]
This is implemented in Cri 2.9.0, released just now.

**Comment by ddfreyne** [comment]
The current implementation is a bit funky (not directly useful), see #57.

**Comment by ddfreyne** [comment]
Fixed in 2.9.1.
